coderd/database/dbgen/generator.go

```diff
@@ -9,13 +9,15 @@ import (
 	"testing"
```

Comment:
nit: Could you please check what's in the upstream? Apparently, import sorting is messed up here.